general-doc/thyra.rst

```diff
@@ -0,0 +1,60 @@
+.. index:: app; decentralized, thyra
+
+.. _general-thyra:
+
+Thyra
+=================
+
+The project
+-------------
+
+Thyra is a secured gate to Massa blockchain and pure decentralization.
+Developped by Massa Labs, this application provides a user-friendly way of accessing,
+using and building on the blockchain while preventing users from possible attack or hack.
+Indeed, because Thyra is installed on your own device there is absolutely nothing
+and no one between you and the blockchain. Security and decentralization are the app's pillars.
+
+Concretely, Thyra enables every user to have a custom experience with the blockchain.
+Once the app installed, users will be able to activate specific functionalities in a plug-and-play mode
+that will provide them what they need. For instance, you will be able to run a distant node in a couple of clicks
+or if you prefer you will be able to create and publish a website on the blockchain.
+These functionalities are, technically speaking, plugins that can be developed by all developers that wish to.
+All details about how you can contribute will be added very soon.
+
+The installation
+-----------------
+
+To install Thyra now, you just have to download the app based on your device.
+
+* Desktop
+
+  * `Windows <https://github.com/massalabs/thyra/releases/latest/download/thyra-installer_windows_amd64.exe>`_
+  * MacOS
+
+    * `ARM <https://github.com/massalabs/thyra/releases/latest/download/thyra-installer_darwin_arm64>`_
+    * `AMD <https://github.com/massalabs/thyra/releases/latest/download/thyra-installer_darwin_amd64>`_
+
+  * `Linux <https://github.com/massalabs/thyra/releases/latest/download/thyra-server_linux_amd64>`_
+
+* Mobile
+
+  * Soon!
+
+The technical details
```

Comment:
Probably not... it was more as a reminder / for people to know we're working on it. Teasing a little bit.
Do you think it's best to remove them?

Reply:
I'd remove them. I don't think it looks very "professional". Also given the number of tasks we have, chances are it will take some time to add it and it can lead to a deceptive effect.

Reply:
Done